website/docs/reference/global-configs/resource-type.md

```diff
@@ -4,7 +4,7 @@ id: "resource-type"
 sidebar: "resource type"
 ---
 
-The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as unit tests) from the `dbt run`, `dbt build`, and `dbt clone` commands.
+The `--resource-type` and `--exclude-resource-type` flags include or exclude resource types (such as [saved queries](/docs/build/saved-queries) and [unit tests](/docs/build/unit-tests)) from the `dbt build` and `dbt clone` commands.
```

Comment:
Is there a reason why we would list only those options?

Also, shouldn't we make it depend on the dbt version? (like unit tests don't exist before 1.8) ; or add some text saying that different dbt versions support different resource types?

Reply:
@b-per great flag! i can add a version block for 1.8+. is there a way i can check what types of flags are available for --resource-type? i tried checking in the cloud cli but i wasn't sure how to get a correct list

Reply:
ok i've added versions, huge thanks!